x-pack/plugins/security_solution/server/lib/detection_engine/rule_actions/deletion_migration/get_throttle.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsFindResult, Logger } from '../../../../../../../../src/core/server';
+import { SideCarAction } from './types';
+
+export interface GetThrottleOptions {
+  actionSideCar: SavedObjectsFindResult<SideCarAction>;
+  logger: Logger;
+}
+
+export const getThrottle = ({ actionSideCar, logger }: GetThrottleOptions): string => {
+  if (actionSideCar.attributes.alertThrottle != null) {
```

Comment:
We should check "rule" first intead of "alert" first since terminology changed and it's more than likely rule that is being stored as a more first class than "alert".